src/renderer/services/file-management-system/index.ts

```diff
@@ -222,60 +229,52 @@ export default class FileManagementSystem {
     onProgress: (uploadId: string, progress: UploadProgressInfo) => void
   ): Promise<void> {
     // Request job from JSS & validate if it is retryable
-    const upload = (await this.jss.getJob(uploadId)) as UploadJob;
+    const fuaUpload = (await this.jss.getJob(uploadId)) as UploadJob;
 
     // Avoid attempting to retry a successful job, should be an update in that case
-    if (upload.status === JSSJobStatus.SUCCEEDED) {
+    if (fuaUpload.status === JSSJobStatus.SUCCEEDED) {
       throw new Error(
-        `Upload cannot be retried if already successful, actual status is ${upload.status}.`
+        `Upload cannot be retried if already successful, actual status is ${fuaUpload.status}.`
       );
     }
 
     // Attempt to resume an ongoing upload if possible before scraping this one entirely
-    const { fssUploadId } = upload.serviceFields;
+    const { fssUploadId } = fuaUpload.serviceFields;
     let resumeError: Error | undefined;
-    if (fssUploadId) {
-      let fssStatus;
-      try {
-        fssStatus = await this.fss.getStatus(fssUploadId);
-      } catch (error) {
-        // No-op: move on if this failed
-      }
 
-      if (
-        fssStatus?.status === UploadStatus.WORKING ||
-        fssStatus?.status === UploadStatus.RETRY ||
-        fssStatus?.status === UploadStatus.COMPLETE
-      ) {
+    if (fssUploadId) {
+      const fssStatus = await this.fss.getStatus(fssUploadId);
```

Comment:
What is the value in the no-op?

Since there is no dependency on it, I see having the `fssStatus` variable at all as dead code.

Reply:
> > Since there is no dependency on it, I see having the fssStatus variable at all as dead code.
> 
> It is used right below this...?

Ok, i was thinking about something else when I wrote that.  Nm.



> > What is the value in the no-op?
> 
> So that if/when we resume we could enter the try/catch and have the upload get canceled if we can't resume. After thinking about this more, I'd just expand the try/catch to include this request.

I'll look at this.

Reply:
Discussed in personf, implented change